### ISSUE TITLE
google search console warning about author.url if user is deleted

### ISSUE THREAD

**Issue by barisusakli** [NodeBB/NodeBB#12689]
![image](https://github.com/NodeBB/NodeBB/assets/2583394/ee3d65a3-c57a-4b23-b76c-950c2c9b895b)

When a user is deleted author field doesnt have url property causing a warning in google search console

https://github.com/NodeBB/nodebb-theme-harmony/blob/main/templates/partials/topic/post.tpl#L18

**Comment by mySebbe** [comment]
Current Harmony markup emits `author` as a `Person`, but omits `author.url` when the deleted user has no `userslug`. There is no truthful profile URL to insert in that state. Should anonymized/deleted authors omit the `Person` object entirely, or should NodeBB define a real canonical “deleted user” route? An invented fragment or unrelated forum URL would only silence the validator with inaccurate structured data.

**Comment by julianlam** [comment]
@barisusakli the line now reads

`{{{ if ./user.userslug }}}<meta itemprop="url" content="{config.relative_path}/user/{./user.userslug}">{{{ end }}}`

Fixed?

**Comment by barisusakli** [comment]
Nope, that's actually what causes the warning, if the user is deleted userslug is empty string and the page doesn't have a url property for the author.
